lib/manageiq/appliance_console/database_admin.rb

```diff
@@ -140,21 +138,22 @@ def ask_ftp_file_options
       end
 
       def ask_custom_file_options(server_uri)
-        @filename    = just_ask(*filename_prompt_args) unless action == :restore
-        sample_case  = server_uri.split("/").last
-        hostname     = URI(server_uri).host
-        uri_filename = ask_custom_prompt(hostname, 'filename', "Target filename (e.g.: #{sample_case})")
-        @uri         = server_uri.gsub(sample_case, uri_filename)
+        hostname  = URI(server_uri).host
```

Comment:
This is minor:

I found this a bit confusing to me for a bit since I saw you parsing out the `host` from the `server_uri`, and thought that was being being assigned to `@uri`.  Turns out that wasn't the case, but we also aren't using the `hostname` in any meaningful fashion except as a key.  Seems like we could just as easily do this:

```yaml
  database_admin:
    menu_order:
    - local
    - ftp://ftp.example.com/incoming/
    local: Local file
    prompts:
      ftp://ftp.example.com/incoming/:
        enabled_for: ["backup", "dump"]
        ...
```

And achieve the same affect by just passing around `server_uri` as the key.

* * * 

Again, this is minor, just thought it was a bit confusing from both a code and configuration perspective since there is two different "keys" being passed around.

Reply:
Lets keep this as the host for now.
Using the full path would simplify the code and may be more clear
But I like reading the short prompt version

Reply:
Works for me.

If the above wasn't clear to on lookers, my above critique is pretty subjective, and I don't hold a hard opinion on this.